aptos_sdk/async_client.py

```diff
@@ -30,12 +30,21 @@
 class ClientConfig:
     """Common configuration for clients, particularly for submitting transactions"""
 
-    expiration_ttl: int = 600
-    gas_unit_price: int = 100
-    max_gas_amount: int = 100_000
-    transaction_wait_in_seconds: int = 20
-    http2: bool = False
-    api_key: Optional[str] = None
```

Comment:
I remove this because we always use it as a instance variables instead of class variables, so this also doesn't break linters. And It's a little weird to define a var as class variable and instance variable at the same time. 😂

Also I think it's easy to see how many settings there are since the constructor is very simple and we can refactor if this is getting large in the future.  What do you think?

Reply:
But ClassVar doesn't change runtime, right? so it can't be used like https://github.com/aptos-labs/aptos-python-sdk/issues/19 asked.
or maybe I didn't get your meaning 😂

Reply:
> I think we might break linters if we remove them.

btw, I think currently mypy doesn't help so much on type checking because:

> "By default mypy will not type check dynamically typed functions. This means that with a few exceptions, mypy will not report any errors with regular unannotated Python" -- [mypy docs](https://mypy.readthedocs.io/en/stable/getting_started.html#dynamic-vs-static-typing)

If we use other tools like pyright or pylance, we may see many type errors :-(
![image](https://github.com/user-attachments/assets/6c4fcf4e-c845-4085-b191-8de01dd22e22)